indico/modules/rb/client/js/common/rooms/RoomEditModal.jsx

```diff
@@ -931,7 +932,21 @@ class RoomEditModal extends React.Component {
                   <Translate>Room has been successfully updated.</Translate>
                 </Message>
                 <Message styleName="submit-message" negative hidden={submitState !== 'error'}>
-                  <Translate>Room could not be updated.</Translate>
+                  <p>
+                    <Translate>Room could not be updated.</Translate>
+                  </p>
+                  <Button
+                    type="button"
+                    icon="undo"
+                    size="mini"
+                    color="red"
+                    content={Translate.string('Reset form')}
+                    onClick={async () => {
+                      // reset the form
+                      this.setState({submitState: ''});
+                      await this.fetchRoomData(roomId);
```

Comment:
I don't think `await`ing has any benefit here, so just calling it (and not making the onClick handler async) sohuld be enough.

Reply:
Actually, you can simply call `fprops.form.reset();` to reset the form without the need for extra requests.